pizza_cutter_metadetect/run_metadetect.py

```diff
@@ -368,6 +379,25 @@ def _do_metadetect(config, mbobs, gaia_stars, seed, i, preconfig, shear_bands):
 
 
 def get_part_ranges(part, n_parts, size):
```

Comment:
I think the set of these part ranges can be generated in a simpler way, e.g.
```python
total = list(range(size))
part_ranges = [total[start:start+chunksize] for start in range(0, size, chunksize)]
```
then you can pull out the part range by index

Reply:
This won't quite be the same. Right now I take the last chunk beyond `size // chunksize` chunks and distribute its entires to the other chunks. I am going to punt on this change for later.

Reply:
no problem, ignore if you want

Reply:
thank you!